Dockerfile

```diff
@@ -25,23 +25,23 @@ COPY . .
 
 # Build the application, with optimizations, with static linking, and using jemalloc
 # N.B.: The static version of jemalloc is incompatible with the static Swift runtime.
-RUN swift build -c release \
+RUN mkdir /staging
```

Comment:
There is a package resolve line up there, i wonder if that can be combined with the `swift build` step since `swift build` is now using caching anyway.